include/exadg/solvers_and_preconditioners/multigrid/multigrid_input_parameters.h

```diff
@@ -293,7 +293,7 @@ struct MultigridData
   {
     print_parameter(pcout, "Multigrid type", enum_to_string(type));
 
-    if(type != MultigridType::hMG || type != MultigridType::hcMG || type != MultigridType::chMG)
+    if(!(type == MultigridType::hMG || type == MultigridType::hcMG || type == MultigridType::chMG))
```

Comment:
This is an actual change to be looked at - the previous comparison always resulted in `true` because when say the first equality is true, the second must be false by definition.

Reply:
I think the logical consequence would be to write a short function `involves_p_transfer()`, given that we already have such functions for h-/c-transfer (see a few lines below).

Reply:
done in PR #52